.github/workflows/ci.yml

```diff
@@ -3,10 +3,14 @@ name: CI
 on:
   push:
     branches: [master]
+    paths-ignore:
+      - "doc/**"
   pull_request:
     branches:
       - master
       - 1.2.x
+    paths-ignore:
+      - "doc/**"
```

Comment:
For this file, we probably don't want to include it (for all others it's fine). Because this file contains the doc build, and we still want to run that if there are doc changes.